test/e2e-ansible/e2e_ansible_suite_test.go

```diff
@@ -57,37 +50,12 @@ var _ = BeforeSuite(func(done Done) {
 	By("creating the repository")
 	Expect(tc.Prepare()).To(Succeed())
 
-	By("checking the cluster type")
-	kubectx, err = tc.Kubectl.Command("config", "current-context")
+	By("getting the cluster Kind")
```

Comment:
Can `fetching the current-context` be a comment for this?

Reply:
We can. It is fine.